goal_src/engine/game/projectiles.gc

```diff
@@ -1287,7 +1285,7 @@
    (let ((a1-8 (new 'stack-no-clear 'collide-edge-hold-list)))
     (set! (-> a1-8 num-allocs) (the-as uint 1))
     (set! (-> a1-8 num-attempts) (the-as uint *touching-list*))
-    (dummy-40 (-> self root-override) a1-8)
+    (dummy-40 (-> self root-override) (the-as vector2w a1-8))
```

Comment:
like earlier, I'm not sure this right, but I don't know what the right answer is.